src/cmds/test.ts

```diff
@@ -53,8 +60,15 @@ export const handler = (
     console.log(chalk.red('NO CODE FOUND'));
     return;
   }
+  if (exactTests){
+    exactTests = exactTests.sort((a,b) => a-b);
+    if (exactTests[0] < 1){
```

Comment:
0 indexed is better, to keep a standard. Currently show and test show results 0 indexed.